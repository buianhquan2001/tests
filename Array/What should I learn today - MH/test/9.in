20
29
12 1 9 1 14 8 15 21 17 8 7 24 17 19 23 1 6 17 8 6 4 20 8 7 17 24 6 3 17 
11
15 21 16 9 13 11 15 15 20 23 17 
1
17 
2
6 17 
7
6 6 17 19 14 23 14 
3
17 9 10 
9
5 21 23 24 6 4 11 25 1 
6
6 12 12 14 9 11 
3
9 20 22 
15
19 15 17 23 2 21 1 15 21 14 23 17 5 16 6 
10
2 6 23 13 23 12 18 15 5 4 
5
12 1 9 1 25 
12
3 15 14 1 14 10 22 14 7 5 20 11 
3
25 23 14 
4
4 15 6 9 
30
14 13 21 8 18 21 1 25 22 22 20 16 24 21 21 21 18 15 25 3 22 14 3 10 17 14 14 22 23 11 
16
6 6 10 1 20 19 23 7 17 20 2 2 17 1 24 9 
28
24 4 4 11 21 22 25 6 10 11 15 24 4 8 20 24 3 8 5 9 20 4 12 17 18 20 12 11 
7
21 4 17 23 9 8 1 
4
13 22 4 22 
