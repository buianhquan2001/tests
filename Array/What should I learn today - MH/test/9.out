english
english
math
english
physics
math
english
physics
physics
physics
english
english
english
math
english
physics
math
math
math
math
